### PR TITLE
History frontend

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Frontend for user history, for both the transfer account and user pages!  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[Audit History Frontend](https://trello.com/c/mFHK7L4N/920-audit-history-frontend)

File: app/client/components/history/historyDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  render() {
+    const stringList = this.props.changes.map(change => {
+      return `${toTitleCase(
+        replaceUnderscores(change.column_name)
+      )} changed from "${change.old_value}" to "${change.new_value}" by ${
+        change.change_by.email
+      }`;
+    });
```
Comment (tristanhcole) [inline app/client/components/history/historyDrawer.tsx:23]: Can we have the email in a teal color?

How hard is it to display the DateTime for each item?

File: app/client/components/user/EditUserForm.tsx [hunk 2]
```diff
@@ -193,11 +197,22 @@ const EditUserForm = (props: Props) => {
...
+                onClick={props.onViewHistory}
+                hidden={
+                  !(
+                    props.adminTier === "superadmin" ||
+                    props.adminTier === "sempoadmin"
+                  )
+                }
+              >
+                View History
+              </Button>
```
Comment (tristanhcole) [inline app/client/components/user/EditUserForm.tsx:215]: I don't think the history button should be "primary" color
I think the button order should be:
  │           delete           │     │         history            │   │             save                │

File: config.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 logging.basicConfig(level=env_loglevel)
 logg = logging.getLogger(__name__)
 
-VERSION = '2.0.5'  # Remember to bump this in every PR
+VERSION = '2.0.6'  # Remember to bump this in every PR
```
Comment (tristanhcole) [inline config.py:8]: Shouldn't have any backend bump.

Should be a minor bump on frontend `2.1.0` in package.json

File: app/client/components/history/historyDrawer.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <Drawer
+        title="Account History"
+        placement="right"
+        visible={this.props.drawerVisible}
+        onClose={this.props.onClose}
+        width={500}
+      >
+        <Timeline>
+          {stringList.map(item => (
+            <Timeline.Item>{item}</Timeline.Item>
```
Comment (tristanhcole) [inline app/client/components/history/historyDrawer.tsx:34]: This item needs a key
Screen Shot 2021-12-15 at 4 11 41 pm

**Review by tristanhcole — COMMENTED** [review]
Overall - looks good to me and works well! Nice work @michielderoos 

There is one small bug to fix

File: app/client/components/history/historyDrawer.tsx (hunk 1 above)
Comment (michielderoos) [reply app/client/components/history/historyDrawer.tsx:23]: Done!

**Review by tristanhcole — APPROVED** [review]
LGTM!

**Pull request merged**